test/e2e/customconfigs/managejobswithoutqueuename_test.go

```diff
@@ -655,7 +657,7 @@ var _ = ginkgo.Describe("ManageJobsWithoutQueueName", ginkgo.Ordered, func() {
 				gomega.Eventually(func(g gomega.Gomega) {
 					g.Expect(k8sClient.Get(ctx, client.ObjectKeyFromObject(sts), createdStatefulSet)).To(gomega.Succeed())
 					g.Expect(createdStatefulSet.Status.ReadyReplicas).To(gomega.Equal(int32(3)))
-				}, util.LongTimeout, util.Interval).Should(gomega.Succeed())
+				}, util.VeryLongTimeout, util.Interval).Should(gomega.Succeed())
```

Comment:
Actually, I think the time is not going to help. We found the issue for scale up, which is likely the same here, ptal: https://github.com/kubernetes-sigs/kueue/issues/6883#issuecomment-3484947580

cc @IrvingMg @mbobrovskyi